src/main/java/sirius/web/controller/Facet.java

```diff
@@ -25,6 +26,8 @@
     private String title;
     private List<String> values;
     private final ValueComputer<String, String> translator;
+    private boolean facetCollapsingEnabled = false;
```

Comment:
redundant - visibleFacectItems controls this

Reply:
It should be possible to completely disable collapsing. To what should visibleFacectItems be set in this case?